src/authentication/views/RegisterOrLogin.tsx

```diff
@@ -92,15 +93,15 @@ const RegisterOrRegisterOrLogin: FunctionComponent<RegisterOrLoginProps> = ({
 										<Spacer margin={['top-small', 'bottom-small']}>
 											<Button
 												block
-												className="c-button-smartschool"
+												className="c-button-smartschool login-with-smartschool"
```

Comment:
is `c-button-smartschool` niet specifiek genoeg om te kunnen targetten in uw tests?